### PR TITLE
hyperterm is now hyper!

### PR DESCRIPTION
**Pull request by patrickleet**

### CODE REVIEW

**Comment by patrickleet** [comment]
Should probably bump the minor version, `1.4.0`?

**Comment by patrickleet** [comment]
README will need to be updated as well

**Comment by patrickleet** [comment]
There were other references to hyperterm that needed to be update as well. Also, fork needed to be updated to fork to `~/.hyper_plugins` instead of `~/.hyperterm_plugins`

**Comment by patrickleet** [comment]
Also updated the tests and package.json

File: package.json [hunk 1]
```diff
@@ -19,6 +19,8 @@
   },
   "repository": "zeit/hpm",
   "keywords": [
+    "hyper.app",
```
Comment (patrickleet) [inline package.json]: Does anyone know if keywords can have a . ?

**Comment by patrickleet** [comment]
list remote still searches for keywords `hyperterm` - not sure best way to proceed here as existing plugins still use `hyperterm` keyword

**Comment by MiracleBlue** [comment]
Let's just search for both for a while, and prompt package authors to update their keywords.   That's the only way to go without 90% of hyper packages suddenly disappearing from search initially, and I'm sure there's no real legal issue with a search keyword, at least during the transition (but IANAL so dont take my word for it).  

Also, regarding allowed characters in keywords, it seems there are no restrictions: https://github.com/npm/npm/issues/7072

File: package.json (hunk 1 above)
Comment (MiracleBlue) [inline package.json]: https://github.com/npm/npm/issues/7072 looks like it :)

**Comment by MiracleBlue** [comment]
Just tried finding a way to search by multiple keywords against the couchdb thing, none of my attempts worked.  Might need someone with more knowledge of couchdb/npm registry to pitch in on how to do that, or we could look at [using npms.io's api](https://api-docs.npms.io/#api-Search-ExecuteSearchQuery) (I use them for searching npm instead of the standard crappy npm search interface, and they support multiple keywords), or if all else fails, we'd have to do 2 separate requests, which might not be too bad in the short term.   We could set a hard cutoff date for package authors to update their keywords by, at which point we just disable the old keyword search.  Lots of options!

**Comment by chrisdwheatley** [comment]
We could create a bot to submit pull requests to repositories to update their keywords, similar to what something like npmcdn-to-unpkg-bot does - https://github.com/chrisdwheatley/glare/pull/2

File: package.json (hunk 1 above)
Comment (leo) [inline package.json]: Use  "hyper" please.

**Comment by leo** [comment]
Keep https://github.com/zeit/hpm/pull/39 in mind 😊

**Comment by patrickleet** [comment]
with #39, I think that's everything - anything else needed for this?

**Comment by leo** [comment]
I'm at home :trollface:

**Comment by chrisdwheatley** [comment]
#39 doesn't seem to actually return results for both 'hyperterm' and 'hyper', just 'hyperterm', see 
https://registry.npmjs.org/-/_view/byKeyword?startkey=[%22hyperterm%22,%22hyper%22]&endkey=[%22hyperterm%22,{}]&group_level=4

Also, if we switch to searching for 'hyper' only then results not related to Zeit's hyper are returned, e.g. https://registry.npmjs.org/-/_view/byKeyword?startkey=[%22hyper%22]&endkey=[%22hyper%22,{}]&group_level=4

**Comment by leo** [comment]
My dream solution would be to search for all of these:

- Keyword "hyperterm"
- Keyword "hyper-plugin"
- Name starts with "hyper-"

**Pull request merged**